### DISCUSSION TITLE
Lora extract in Supermerger. Difussers downgrade problem

### DISCUSSION THREAD

**Discussion by tottem** [hako-mikan/sd-webui-supermerger#247] [Q&A]
Hi. 
Trying to use supermerger automatic 1111 extension to extract lora from custom dreambooth model. From the GitHub page of the extension is necessary to use difussers less than 0.14. 
The problem is that I'm using automatic 1111 V1.6 version and I can't find any information about downgrade diffusers. 
Thanx. 
Sorry for my English

**Comment by yishiliuc** [comment]
did you use the extension dreambooth? there is a requirement txt file in dreambooth extension folders,you can change the version in that file

**Reply by tottem** [reply]
Thanx... I use to train the dreambooth model with the Shivam Script in Google colab...

https://colab.research.google.com/github/ShivamShrirao/diffusers/blob/main/examples/dreambooth/DreamBooth_Stable_Diffusion.ipynb#scrollTo=BzM7j0ZSc_9c

then you mean that a I have to choose the difussers versión when training... sorry I didn't know that... I don´t understand the whole concept of training... is there any way to change the version of the trained model or I have to train again choosing the right version... 
I can't find in the Google colab from Shivam the way to choose the diffusers version when training.
Thanx in advance.
